README.md

```diff
@@ -32,7 +34,7 @@ import '@ng-web-apis/universal/mocks';
 
 ## Tokens
 
--   `WINDOW` — no need to provide fallback, Angular Universal handles it
+-   `WINDOW` — add `UNIVERSAL_WINDOW` to provide type-safe mock object, effectively mocking all `navigator` related entities
```

Comment:
let's add a sample of how to provide UNIVERSAL_WINDOW and add a reference to it here. And some with brief description of use cases